src/test/java/seedu/address/model/DataTest.java

```diff
@@ -49,52 +49,52 @@ public void resetData_withDuplicatePersons_throwsDuplicatePersonException() {
         Person editedAlice = new PersonBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
```

Comment:
Does our person class have an **Address**, and we wont be using tags right?

Reply:
this will be changed when @HoWeiChin merges the changes for the person